database/sql/src/main/scala/cromwell/database/slick/SlickDatabase.scala

```diff
@@ -100,6 +100,8 @@ class SlickDatabase(override val originalDatabaseConfig: Config) extends SqlData
     actionThreadPool, database.executor.executionContext.reportFailure
   )
 
+  protected[this] lazy val insertBatchSize = databaseConfig.as[Option[Int]]("database.insert-batch-size").getOrElse(2000)
```

Comment:
someone got caught saying "how could this possibly break what I had previously?"